docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
-
-        ```bash
-        geth attach new-node-1/geth.ipc
-        ```
-
-        Once connected in geth console, run:
-
-        ```javascript
-        eth.accounts
-        ```
-
-        List of the existing accounts is displayed:
+       ]
+   },
+   "alwaysSendTo": []
+}
+```
 
-        ```json
-        ["0x23214cd88f46865207fa1d2a69971a37cdbf526a"]
-        ```
+### 6. Start Tessera 0
 
-        Unlock the account:
+In the `Tessera-0` directory, start Tessera 0.
 
-        ```javascript
-        personal.unlockAccount("0x23214cd88f46865207fa1d2a69971a37cdbf526a");
-        ```
+```bash
+java -jar <path to Tessera>/tessera.jar -configfile config.json
+```
 
-        Type in your passphrase when asked for:
+### 7. Start Tessera 1
 
-        ```text
-        Unlock account 0x23214cd88f46865207fa1d2a69971a37cdbf526a
-        Passphrase:
-        ```
+In the `Tessera-1` directory, start Tessera 1.
 
-        If operation succeeded, result is:
+```bash
+java -jar <path to Tessera>/tessera.jar -configfile config.json
+```
 
-        ```text
-        true
-        ```
+### 8. Start GoQuorum node 0
 
-    Load the contract script, deploy it on the blockain:
+In the `Node-0` directory, start start GoQuorum node 0 specifying the Tessera 0 node to attach to.
 
-    ```javascript
-    loadScript("private-contract.js")
-    ```
+```bash
+PRIVATE_CONFIG=/<path to IBFT-network>/IBFT-network/Tessera-0/tm.ipc geth --datadir data --nodiscover --istanbul.blockperiod 5 --syncmode full --mine --minerthreads 1 --verbosity 5 --networkid 10 --rpc --rpcaddr 127.0.0.1 --rpcport 22000 --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul --emitcheckpoints --port 30300 --allow-insecure-unlock
+```
 
-    Result:
+!!! caution
+    The `--allow-insecure-unlock` option enables insecure account unlocking for educational purposes
+    only. In production environments, ensure account keys are secured appropriately.
 
-    ```text
-    Contract transaction send: TransactionHash: 0x7d3bf7612ef10c71f752e881648b7c8c4eee3223acab151ee0652447790836a6 waiting to be mined...
-    true
-    Contract mined! Address: 0xe975e1e11c5268b1efcbf39b7ee3cf7b8dc85fd7
-    ```
+### 9. Start GoQuorum node 1
 
-    Congratulations! You **successfully** sent a private transaction from node 1 to node 2!
+In the `Node-1` directory, start start GoQuorum node 1 specifying the Tessera 1 node to attach to.
 
-    !!! note
-        if you do not have a valid public key in the array in private-contract.js, you will see the following error when the script in loaded.
+```bash
+PRIVATE_CONFIG=/<path to IBFT-network>/IBFT-network/Tessera-1/tm.ipc geth --datadir data --nodiscover --istanbul.blockperiod 5 --syncmode full --mine --minerthreads 1 --verbosity 5 --networkid 10 --rpc --rpcaddr 127.0.0.1 --rpcport 22001 --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,istanbul --emitcheckpoints --port 30301
```

Comment:
Ask for replacement of the placeholder `<path to IBFT-network>` in the cmd.

Reply:
Fixed